File: src/main/java/wordseeker/gui/InputField.java
package wordseeker.gui;

import javax.swing.*;
import java.awt.*;

/**
 * User: rad1kal
 * Date: 15.03.13
 */
class InputField extends JTextField {

    InputField() {
        super();
        setBackground(Color.DARK_GRAY);
        setFont(new Font("Verdana", Font.BOLD, 14));
        setForeground(Color.LIGHT_GRAY);
        Dimension d = new Dimension(200, 30);
        setPreferredSize(d);
        setMinimumSize(d);
        setBorder(null);
        setCaretColor(Color.white);
    }
}

File: src/main/java/wordseeker/gui/InputField.kt
package wordseeker.gui

import javax.swing.*
import java.awt.*

/**
 * User: rad1kal
 * Date: 15.03.13
 */
internal class InputField : JTextField() {

    init {
        background = Color.DARK_GRAY
        font = Font("Verdana", Font.BOLD, 14)
        foreground = Color.LIGHT_GRAY
        val d = Dimension(200, 30)
        preferredSize = d
        minimumSize = d
        border = null
        caretColor = Color.white
    }
}
